lib/capn.c

```diff
@@ -7,6 +7,8 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
+#pragma GCC diagnostic ignored "-Wmissing-field-initializers"
```

Comment:
Is this still neccessary? I thought the previous PR fixed the missing initializers?
(If yes, please put it inside `#ifdef __GNUC__`)